.github/workflows/kcp-image.yaml

```diff
@@ -7,13 +7,14 @@ on:
   push:
     branches:
       - main
+      - 'release-*'
 
 jobs:
   kcp-image:
     if: github.repository_owner == 'kcp-dev'
     name: Build and Publish KCP Image
+    environment: unstable
```

Comment:
Is this going to have different values in different branches?

Reply:
yes

Reply:
Thanks